src/components/drawing/index.js

```diff
@@ -579,16 +579,17 @@ drawing.makeTester = function(gd) {
         tester.node()._cache = {};
     }
 
-    gd._tester = tester;
-    gd._testref = testref;
+    drawing.tester = gd._tester = tester;
+    drawing.testref = gd._testref = testref;
```

Comment:
:stop_sign:  Hold the presses :stop_sign: 

Caching the tester div and testref path on the `Drawing` module object will make plotly.js even more broken in child windows than it is at the moment. From https://github.com/plotly/plotly.js/issues/702 and the attempted PR https://github.com/plotly/plotly.js/pull/764, plotly.js CSS is  inserted once per `<html>` page - which doesn't propagate to child windows. We should instead insert plotly.js CSS once per `document`.

Similarly, we should be caching `tester` and `testref` once per `document` too. That is, child windows need their own tester nodes. As we don't have access to cache once-per-document object (adding refs to `document` sounds dirty), caching `tester` and `testref` on the `gd` might actually be the best we can do. 

This would imply passing `gd` to `Drawing.bBox` below - which isn't a high price to pay.